lib/TableGen/GenDialect.cpp

```diff
@@ -298,6 +301,18 @@ void llvm_dialects::genDialectDefs(raw_ostream& out, RecordKeeper& records) {
       return s_key;
     }
 
+    bool $Dialect::isDialectOp(::llvm::CallInst& op) {
+      return isDialectOp(op.getCalledFunction()->getName());
```

Comment:
This should maybe check that `op.getCalledFunction() != nullptr` (which happens for indirect calls).

Reply:
https://github.com/GPUOpen-Drivers/llvm-dialects/pull/96